thirdparty/libltdl/Makefile

```diff
@@ -7,7 +7,7 @@ pname := libltdl
 uname := libtool
 pfile := $(uname)-$(pvers).tar.gz
 psrc_file := $(SRC_DIR)/$(pfile)
-purl := http://reflection.oss.ou.edu/gnu/gnu/libtool/$(pfile)
+purl := http://ftpmirror.gnu.org/libtool/$(pfile)
```

Comment:
I know https://www.gnu.org/server/mirror.html says use http://ftpmirror... but it appears they do have/support https so please change the URL here to be https://...